jarviscli/CmdInterpreter.py

```diff
@@ -52,6 +52,7 @@ def __init__(self, first_reaction_text, prompt, first_reaction=True, enable_voic
                         "chat",
                         {"check": ("ram", "weather", "time", "forecast")},
                         "chuck",
+                        "clear_scr",
```

Comment:
No error, but we have to add "clear" in actions tuple because we have defined "do_clear". It does not matter what you add in tuple, but just for good code to read